tests/test_project.py

```diff
@@ -1250,8 +1250,12 @@ def test_main_run(self):
     def test_main_next(self):
         assert len(self.project)
         job_ids = set(self.call_subcmd("next op1").decode().split())
+        assert len(job_ids) > 0
         even_jobs = [job.get_id() for job in self.project if job.sp.b % 2 == 0]
         assert job_ids == set(even_jobs)
+        # Use only exact operation matches
+        job_ids = set(self.call_subcmd("next op").decode().split())
+        assert len(job_ids) == 0
```

Comment:
@bdice I am in favour of this test. I wrote it myself but chose not to push it because it is kind-of misleading (It may be possible that `op` is a valid operation and its just the case that there are no eligible jobs). I think we might want to add a test which would match the exact output thrown by the `next` command-line flag after #442 is merged.

Reply:
> (It may be possible that op is a valid operation and its just the case that there are no eligible jobs).
This is only going to run on the test project that we define -- if someone adds an operation named `op` then this test might fail, but otherwise this is a good test.